server/run-delivery.sh

```diff
@@ -42,7 +42,7 @@ OUTPUT_IMAGE=registry.centos.org/${TARGET_NAMESPACE}/${TO}
 #sleep 20
 
 _ "Send the image details to test_tube for testing"
-python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE} ${NOTIFY_EMAIL}
+python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE}
```

Comment:
No need to explicitly mention the notify email address since it is already in the content present over tube.